README.md

```diff
@@ -1035,7 +1034,7 @@ Collaborative software or [groupware](https://en.wikipedia.org/wiki/Collaborativ
 - [BlueMind](https://www.bluemind.net/en/) - Groupware with email, calendar, addressbooks, exchange active sync, exchange MAPI protocol support. ([Source Code](https://forge.bluemind.net/stash/projects/BM/repos/bluemind-public/browse)) `AGPL-3.0` `Java`
 - [Citadel](https://www.citadel.org/) - Groupware including email, calendar/scheduling, address books, forums, mailing lists, IM, wiki and blog engines, RSS aggregation and more. ([Source Code](https://www.citadel.org/source.html)) `GPL-3.0` `C`
 - [Corteza](https://cortezaproject.org) - CRM including a unified workspace, enterprise messaging and a low code environment for rapidly and securely delivering records-based management solutions. ([Demo](https://latest.cortezaproject.org), [Source Code](https://github.com/cortezaproject/corteza-server)) `Apache-2.0` `Go`
-- [Cozy Cloud](https://cozy.io) - Personal cloud where you can manage and sync your contact, files and calendars, and manage your budget with an app store full of community contributions. ([Source Code](https://github.com/cozy)) `GPL-3.0` `Nodejs`
+- [Cozy Cloud](https://cozy.io/en/) - Personal cloud where you can manage and sync your contact, files and calendars, and manage your budget with an app store full of community contributions. ([Source Code](https://github.com/cozy/cozy-drive)) `GPL-3.0` `Nodejs`
```

Comment:
Please don't force the language on links, many websites use language detection in browser. Those links blocks that and website seems to be english readers only.
Thanks

Reply:
I wasn't aware browsers did that with those links.

Although the main none language link will redirect to the /en/ version. Adding the /en/ just stops that redirection directly.

Will remove though!

Reply:
I understood you used hecat to catch that there was a automatic redirection, but in these cases (languages) they are useful for the user :)
Thank you !